packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js

```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.storages.cold-archive', {
+    url: '/cold-archive',
+    component: 'ovhManagerPciProjectStorageColdArchive',
+    /* onEnter: /!* @ngInject *!/ (pciFeatureRedirect) => {
+      return pciFeatureRedirect(PCI_FEATURES.PRODUCTS.COLD_ARCHIVE);
+    }, */
```

Comment:
Why is commented. Maybe put a comment to explain this.

Reply:
We can remove it for the moment.